career-paths/activity-specialized-fields-jigsaw.md

```diff
@@ -0,0 +1,90 @@
+# Activity: Specialized Fields Jigsaw
+
+## Goal
+
+Our goal is to learn and teach each other about different specialized fields within computing.
+
+We should come out of this activity seeing the breadth and diversity of different specialized fields in computing. This activity should inspire us and show us just a few different options of career development.
+
+## Preparation and Format
+
+In this activity, we will be learning and then teaching each other. We will have two rounds: the research round and the teaching round.
+
+In the research round, you will meet with your first assigned group to learn about an assigned topic.
+
+In the teaching round, your research round group will disseminate and you will go into your second assigned group. It is your responsibility to teach this group about that topic.
```

Comment:
"assigned" twice in such close proximity sounds redundant
"disseminate" is transitive (it needs a direct object to denote what is being spread (esp. information)
more typical to "get into" groups
```suggestion
In the research round, you will meet with your first assigned group to learn about a specified topic.

In the teaching round, your research round group will disband and you will get into your second assigned group. It is your responsibility to teach this group about that topic.
```